observability/observabilitytest/observabilitytest_test.go

```diff
@@ -58,13 +58,42 @@ func TestCheckValueViewReceiverViews(t *testing.T) {
 	}
 }
 
-func TestCheckValueViewExporterViews(t *testing.T) {
+func TestCheckValueViewMetricsReceiverViews(t *testing.T) {
+	doneFn := observabilitytest.SetupRecordedMetricsTest()
+	defer doneFn()
+
+	receiverCtx := observability.ContextWithReceiverName(context.Background(), receiverName)
+	observability.RecordMetricsForMetricsReceiver(receiverCtx, 17, 13)
+	// Test expected values.
+	if err := observabilitytest.CheckValueViewReceiverReceivedTimeSeries(receiverName, 17); err != nil {
```

Comment:
Would be shorter and more consistent to use require.Nilf as in the other tests.

Reply:
Done. Changed it in the entire observability package.